src/Microsoft.Azure.SignalR.Common/ServiceConnections/ManagementServiceConnectionContainer.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.SignalR.Common.ServiceConnections
+{
+    internal class ManagementServiceConnectionContainer : WeakServiceConnectionContainer
+    {
+        private bool _manuallyStop;
```

Comment:
`volatile` needed? `StartCoreAsync` and `StopAsync` may be invoked in different threads?

Reply:
it can be modified in `StopAsync` only, and read by `StartCoreAsync` only